src/renderer/components/dialogs/ForwardMessage/index.tsx

```diff
@@ -45,15 +45,14 @@ export default function ForwardMessage(props: Props) {
     const chat = await BackendRemote.rpc.getFullChatById(accountId, chatId)
     onClose()
     if (!chat.isSelfTalk) {
-      selectChat(accountId, chat.id)
```

Comment:
this is there because then you see the chat you are going to forward to, to prevent user mistakes.

Reply:
Ok - that should be worth a line of an explaining comment in the code :-)

So I have to find why the already opened chat is not scrolling down I guess

Reply:
maybe calling select chat again could already work. or whatever happens when you click again on the chat in the chat-list if you have already selected the chat,